### PR TITLE
Added serializer flexible entity normalizer

### PR DESCRIPTION
**Pull request by SebLours**

### CODE REVIEW

File: sources/lib/Serializer/Normalizer/FlexibleEntityNormalizer.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+        return $object->extract();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsNormalization($data, $format = null)
+    {
+        return $data instanceof FlexibleEntity;
```
Comment (tlode) [inline sources/lib/Serializer/Normalizer/FlexibleEntityNormalizer.php]: I would suggest to check for `\PommProject\ModelManager\Model\FlexibleEntity\FlexibleEntityInterface`

Comment (chanmix51) [inline sources/lib/Serializer/Normalizer/FlexibleEntityNormalizer.php]: :+1:

Comment (SebLours) [inline sources/lib/Serializer/Normalizer/FlexibleEntityNormalizer.php]: Fixed

**Comment by chanmix51** [comment]
Doesn't it need to have a dependency on [Symfony’s serializer](https://packagist.org/packages/symfony/serializer) component ?

**Comment by SebLours** [comment]
For me no, the normalizer can be used by anyone need it with is own symfony serializer.
But i can be wrong ...

**Comment by sanpii** [comment]
@SebLours you add a dependency to `Symfony\Component\Serializer\Normalizer\NormalizerInterface` class. I think add sf serializer to requirement is a good idea.

**Comment by sanpii** [comment]
Thank you :smiley:

**Pull request merged**